### PR TITLE
An Apache2 reverse proxy config

### PR DESCRIPTION
**Pull request by workybee**
Still using apache2 with the dockerized version of drone so needed to figure out a working apache2 config to do the reverse proxying.

### CODE REVIEW

File: content/setup/proxy_reverse.md [hunk 1]
```diff
@@ -45,3 +45,35 @@ location / {
...
 }
 ```
+
+# Apache2
+
+Example Apache2 reverse proxy configuration:
+
+```
+<VirtualHost *:80>
+
```
Comment (tboerger) [inline content/setup/proxy_reverse.md]: Drop this line

File: content/setup/proxy_reverse.md [hunk 2]
```diff
@@ -45,3 +45,35 @@ location / {
...
+	ServerName drone.example.com
+	ServerAlias drone
+
+	ServerAdmin webmaster@localhost
+
+	ErrorLog ${APACHE_LOG_DIR}/drone-error.log
+	CustomLog ${APACHE_LOG_DIR}/drone-access.log vhost_combined
+
+	#-#ProxyPass / http://127.0.0.1:8080/
```
Comment (tboerger) [inline content/setup/proxy_reverse.md]: Drop this?

File: content/setup/proxy_reverse.md [hunk 3]
```diff
@@ -45,3 +45,35 @@ location / {
...
+	#-#ProxyPassReverse / http://127.0.0.1:8080/
+	ProxyPass         / http://localhost:8080/ nocanon
+	ProxyPassReverse  / http://localhost:8080/
+	RequestHeader set X-Forwarded-Proto "http"
+	RequestHeader set X-Forwarded-Port "80"
+	ProxyRequests     Off
+	#ProxyRequests     On	#not needed for reverse proxy
```
Comment (tboerger) [inline content/setup/proxy_reverse.md]: Drop this?

File: content/setup/proxy_reverse.md (hunk 2 above)
Comment (workybee) [inline content/setup/proxy_reverse.md]: not needed for sure, but sometimes localhost doesn't resolve same as 127.0.0.1 so left it in.
I'm not attached.

File: content/setup/proxy_reverse.md (hunk 3 above)
Comment (workybee) [inline content/setup/proxy_reverse.md]: Pulled this detail from the apache2 man page:
   https://httpd.apache.org/docs/current/mod/mod_proxy.html#forwardreverse
after thinking it was needed.
Happy to elide it.

File: content/setup/proxy_reverse.md (hunk 1 above)
Comment (workybee) [inline content/setup/proxy_reverse.md]: dropped

**Comment by tboerger** [comment]
LGTM

**Comment by gtaylor** [comment]
The reverse proxies now require websockets. If you wanted to figure something out and test with 0.5, here's something to get started with: https://github.com/drone/docs/blob/release/0.5/content/installation/network/proxy_reverse.md

**Pull request closed without merging**